docs/doc1.md

```diff
@@ -260,42 +259,49 @@ And remember lists can be nested too.
 $ brun '(f (f (r 1)))' '((70 80) (90 100) (110 120))'
 90
 
-$ run '(f (f (r 1)))' '((70 80) ((91 92 93 94 95) 100) (110 120))'
+$ run '(f (f (r @)))' '((70 80) ((91 92 93 94 95) 100) (110 120))'
 (91 92 93 94 95)
 ```
 
 These environment variables can be used in combination with all other operators.
 
 ```lisp
-$ run '(+ (f 1) (q 5))' '(10)'
+$ run '(+ (f @) (q . 5))' '(10)'
 15
 
-$ run '(* (f 1) (f 1))' '(10)'
+$ run '(* (f @) (f @))' '(10)'
 100
 ```
 
 This program checks that the second variable is equal to the square of the first variable.
 
 ```lisp
-$ run '(= (f (r 1)) (* (f 1) (f 1)))' '(5 25)'
+$ run '(= (f (r @)) (* (f @) (f @)))' '(5 25)'
 1
 
-$ run '(= (f (r 1)) (* (f 1) (f 1)))' '(5 30)'
+$ run '(= (f (r @)) (* (f @) (f @)))' '(5 30)'
 ()
 ```
 
 ## Accessing Environmental Variables Through Integers
 
 In the above examples we were using `run`, calling the higher level language, instead of `brun` for the lower level language.
-This is because for the sake of minimalism in the lower level CLVM language, we address the solution with evaluated integers.
+This is because for the sake of minimalism in the lower level CLVM language, we use self-eval on integers to mean "look up this argument in the environment tree", wheras in `run`, numbers evaluate to themselves.
 
-Calling `1` accesses the root of the tree and returns the entire solution list.
+In clvm assembly (`brun`), calling `1` accesses the root of the tree and returns the entire solution list.
 
 ```lisp
 $ brun '1' '("example" "data" "for" "test")'
 ("example" "data" "for" "test")
 ```
 
+To do the same in ChiaLisp (`run`), use `@`:
+
+```lisp
+$ run '@' '("example" "data" "for" "test")'
+("example" "data" "for" "test")
+```
+
 After that, you can imagine a binary tree of `f` and `r`, where each node is numbered.
```

Comment:
Probably need some more clarification of this now that we have added another paragraph above